### DISCUSSION TITLE
Welcome to the Potter DB Discussions!

### DISCUSSION THREAD

**Discussion by danielschuster-muc** [danielschuster-muc/potter-db#165] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions about Potter DB.
  * Share ideas or suggest new features.
  * Engage with other community members by answering question and help them. 
  * Show what you did with Potter DB.

**Comment by Weranika** [comment]
Hello, something is going wrong: filter and sorting return 500 error, when i try get reques like https://api.potterdb.com/v1/characters?sort=-name or https://api.potterdb.com/v1/characters?filter[name_cont_any]=har
